README.md

```diff
@@ -49,6 +49,24 @@ Given the locking on the **model** there will be some parallelization, but also
 
 Because **Zync** will keep a **log** of **events** it will be able to replay changes and recover the last state just taking last revisions of each **model** and even remove the ones that have been created before but have been deleted.
 
+## Running locally
+
+### Run a PostgreSQL server
+
+It is easy with Docker or Podman
+```
+sudo dnf install postgresql pspg
+docker run -d -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_DB=zync --name postgres10-zync docker.io/circleci/postgres:10.5-alpine
+```
+
+### Start Zync
+```
+export DATABASE_URL=postgresql://postgres:@localhost:5432/zync # or use .env
```

Comment:
I'd rather avoid `.env`

I personally use envrc with direnv but I do not want people to use that unless they know what they are doing